packages/uploadthing/src/internal/handler.ts

```diff
@@ -24,6 +34,39 @@ export const fileSizeToBytes = (input: string) => {
   return Math.floor(bytes);
 };
 
+const fileCountLimitHit = (
+  files: string[],
+  routeConfig: ExpandedRouteConfig
+) => {
+  const counts = {
```

Comment:
```suggestion
  const counts: Record<AllowedFileTypes, number> = {
```

to make sure we keep it sync when adding support for more filetypes